### PR TITLE
protectsite on tnoteswiki and protectsite config per T2003

### PR DESCRIPTION
**Pull request by The-Voidwalker**

### CODE REVIEW

File: LocalSettings.php [hunk 1]
```diff
@@ -5708,6 +5709,19 @@
...
+	// Protect site config
+	'wgProtectSiteLimit' => array(
+		'default' => '1 week',
+		'tnoteswiki' => 'indefinite',
+	),
+	'wgProtectSiteDefaultTimeout' => array(
+		'default' => '1 hour',
+		'tnoteswiki' => '2 hours',
+	),
+	'wgProtectSiteExempt' => array(
```
Comment (Reception123) [inline LocalSettings.php]: Since this is the default, why do we have to define it?

Comment (The-Voidwalker) [inline LocalSettings.php]: No idea, should I remove?

Comment (labster) [inline LocalSettings.php]: I'm not sure if we must have these values, but if we have custom values for extension variables, I'd prefer that we have the default setting listed too.

Comment (Reception123) [inline LocalSettings.php]: Unless a wiki wants a different value than the default, I'm not sure why we should be listing the default. That means we'd have to list all the default values for all the extensions.

Comment (Reception123) [inline LocalSettings.php]: It's not a big deal if we do define the default, of course, just don't see why this should be taking up more space

**Comment by Reception123** [comment]
@The-Voidwalker Sorry for the confusion. I didn't mean remove those defaults, those have to stay! I mean't removing the 'wgProtectSiteExempt' variable only.

**Pull request merged**